Week1/homework/js-exercises/10_compareArrays.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+let firstArray = ["Messi",1,2,3,4];
+let secondArray = ["Ronaldo",5,6,7,8,9,10];
+
+console.log("firstArray's length is",firstArray.length);
+console.log("secondArray's length is",secondArray.length);
+if(firstArray.length === secondArray.length){
+    console.log("They are the same!");
+};
+console.log("Two different sizes");
```

Comment:
This need to be wrapped in else statement, otherwise it will be always printed, even if the arrays are the same size

```
if(firstArray.length === secondArray.length){
    console.log("They are the same!");
}
else {
    console.log("Two different sizes");
}
```